### PR TITLE
Turned exercise 015 into Practice.

### PR DESCRIPTION
**Pull request by divyankachaudhari**
Addresses #2198. Turns [Exercise 15](https://github.com/divyankachaudhari/ocaml.org/blob/010/data/exercises/015_replicate_elements.md) into Practice. 

`dune build --root .` works.

@cuihtlauac Could you please review this? Thank you.

### CODE REVIEW

File: practice/015/answer/impl.ml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let replicate list n =
+  let rec prepend n acc x =
+    if n = 0 then acc else prepend (n-1) (x :: acc) x in
+  let rec aux acc = function
+    | [] -> acc
+    | h :: t -> aux (prepend n acc h) t in
+  (* This could also be written as:
+      List.fold_left (prepend n) [] (List.rev list) *)
+  aux [] (List.rev list);;
```
Comment (cuihtlauac) [inline practice/015/answer/impl.ml]: Watch for the end of file without newline

File: practice/015/ex.ml
Comment (cuihtlauac) [inline practice/015/ex.ml:1]: Add a test with `replicate ["a"; "b"; "c"] 1`

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: practice/015/answer/impl.ml (hunk 1 above)
Comment (divyankachaudhari) [reply practice/015/answer/impl.ml]: Ah yes, done.

File: practice/015/ex.ml
Comment (divyankachaudhari) [reply practice/015/ex.ml:1]: Done.

**Review by cuihtlauac — APPROVED** [review]
Looks good to me. Thanks @divyankachaudhari

**Pull request merged**